### ISSUE TITLE
MySQL Keys with length

### ISSUE THREAD

**Issue by azapps** [JSQLParser/JSqlParser#108]
If I create a Key with a length in MySQL the Parser can not parse it :(

``` SQL
CREATE TABLE animals (
info mediumtext NOT NULL,
KEY idx_info (info(10)));
```

**Comment by wumpz** [comment]
What is the meaning of this? First 10 characters are indexed?

**Comment by azapps** [comment]
Yes, see [here](http://dev.mysql.com/doc/refman/5.0/en/create-index.html):
```
index_col_name:
    col_name [(length)] [ASC | DESC]
```

**Comment by wumpz** [comment]
long way, but now its solved (1.3-SNAPSHOT)
